client/component/main.d.ts

```diff
@@ -1,3 +1,7 @@
+import { ColorMap } from '@h5web/lib/dist';
```

Comment:
Replace with `type ColorMap = import('@h5web/lib').ColorMap;` and similar for `Aspect`